src/style.css

```diff
@@ -0,0 +1,142 @@
+/* stylelint-disable shorthand-property-no-redundant-values */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: content-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+.input {
+  position: relative;
+  height: 100%;
+  width: calc(100% - 60px);
+  font-size: 16px;
+  font-weight: 300;
+  left: 16px;
+  top: -1px;
+  outline: none;
+  border: none;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+.image {
+  position: relative;
+  top: 3px;
+  left: 5px;
+}
+
+.search-bar {
+  position: relative;
+  margin: 20px 8px;
+  padding: 20px;
+  font-size: 24px;
+  height: 28px;
+  max-height: 70px;
+  line-height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.input::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
```

Comment:
Move repeatable colors into variables

Reply:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)